extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1547,6 +1555,31 @@ export class AmpA4A extends AMP.BaseElement {
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
 
+  tryExecuteRealTimeConfig_() {
+    if (!!AMP.RealTimeConfigManager) {
+      try {
+        this.realTimeConfigManager_ = new AMP.RealTimeConfigManager(
```

Comment:
No need to store as a field variable given its only referenced within this function